gcp_variant_transforms/libs/bigquery_schema_to_vcf_header.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
Any reason for not using the existing `bigquery_vcf_schema.py` file? It seems natural to have both `generate_header_fields_from_schema` and `generate_schema_from_header_fields` in one file.
I think we should also have a unit test with header->schema->header to make sure all the assumptions are met and no fields are dropped/added (you can choose the original header to be compatible with 'best effort' reconstruction so that you can exactly match on the reconstructed header). This will also catch things like integer vs. string in the Number field.

Reply:
Very nice advice! Updated. The combined test of header->schema->header is awesome!